histonets/api.py

```diff
@@ -397,3 +401,38 @@ def histogram_palette(histogram, n_colors=8, method='auto', sample_fraction=5,
         sampled_histogram, method=method,
         n_colors=n_colors, background_value=background_value,
         background_saturation=background_saturation)
+
+
+@image_as_array
+def extract_edges(image, regions, simplification_method=None,
+                  simplification_tolerance=1.0):
+    """Build a graph from edges expressed in a binary grid (0's representing
+    holes, 1's paths), and vertices as the center of regions expressed by the
+    the top-left corner and the down-right corner in (x, y) pixels coordinates
+    from the grid."""
+    if simplification_method == 'vw':
+        simplify_func = simplify_coords
```

Comment:
Should these be reversed? It seems like you're checking for vw, if it's set, use the basic simplify_coords. And if it's not, then use simplify_coords_vw.  If the simplification method is vw, should you use the simplify_coords_vw?